crates/consensus/common/src/validation.rs

```diff
@@ -36,6 +36,51 @@ pub fn validate_header_base_fee(
     }
     Ok(())
 }
+// EIP-4895: Beacon chain push withdrawals as operations
+#[inline]
+fn validate_shanghai_withdrawals(block: &SealedBlock) -> Result<(), ConsensusError> {
```

Comment:
we should make these `pub` so we can reuse them across consensus impls:) also make sure they are available from the crate root, i.e. make sure they are exported somehow in `src/lib.rs`